assets/src/components/form/select/model-selects/event.js

```diff
@@ -65,6 +65,7 @@ export default class EventSelect extends Component {
 			...this.props,
 			...selectOpts,
 			...this.state,
+			optionsEntityMap,
```

Comment:
I don't follow why this was needed when it should already be included with the `...this.props` spread...

Reply:
I don't understand how it would be part of the `...this.props` spread when it is a constant that is declared at the top of the file just below the imports:

```js
/**
 * External imports
 */
import { __ } from '@eventespresso/i18n';
import { Component } from '@wordpress/element';
import PropTypes from 'prop-types';

const optionsEntityMap = {
	default: {
		value: 'EVT_ID',
		label: 'EVT_name',
	},
};

/**
 * Select Component for the Event Model.
 */
export default class EventSelect extends Component {
	state = {
		modelName: model.MODEL_NAME,
	};
```